Userland/Libraries/LibJS/Runtime/ErrorTypes.h

```diff
@@ -119,7 +119,7 @@
     M(ObjectPrototypeWrongType, "Prototype must be an object or null")                                                                  \
     M(OptionIsNotValidValue, "{} is not a valid value for option {}")                                                                   \
     M(OutOfMemory, "Out of memory")                                                                                                     \
-    M(OverloadResolutionFailed, "Overload resolution failed")                                                                           \
+    M(OverloadResolutionFailed, "Overload resolution failed on function {}")                                                            \
```

Comment:
Please grep for other usages of this error type - there's at least one more in  IDLGenerators.cpp that needs to be updated.
https://github.com/serenityos/serenity/blob/be8489fb04b1c4d297e44725c27bbf8e623212e8/Meta/Lagom/Tools/CodeGenerators/LibWeb/BindingsGenerator/IDLGenerators.cpp#L2321-L2322